src/mvd-tool.cpp

```diff
@@ -99,6 +98,27 @@ void print_csv(const std::string & filename){
     }
 }
 
+void print_tsv(const std::string & filename){
+    using namespace TSV;
+
+    TSVFile mecombo_file(filename);
+    std::cout << "morph_name\tlayer\tfullmtype\tetype\temodel\tcombo_name\tthreshold_current\tholding_current" << std::endl;
+    for(auto it: mecombo_file.getTSVInfo()) {
+        std::cout << it.morphName << "\t" << it.layer << "\t" << it.fullMType << "\t" << it.eType << "\t" << it.eModel << "\t" << it.comboName << "\t" << it.thresholdCurrent  << "\t" << it.holdingCurrent << std::endl;
```

Comment:
line length should stay below 90 chars
There should be a .clang-format file. If not please use the one from coding-conventions.

Reply:
I'm wondering if we couldn't use some range-based feature to have the equivalent of Python's `join` here? I've used range-v3 for splitting, but it seems a little heavy as a dependency.

Reply:
@ferdonline I run clang-format in all the headers/sources and there are a lot of differences. Should I add them in this PR or maybe it's better to create a new one with the addition of `clang-format` in cmake and the whole project formated?
@matz-e I found something with boost [here](https://stackoverflow.com/a/17660354/11956455) but I am not sure if it is worth it.
Another solution I thought to make this part a bit cleaner is to overload the `<<` operator for the `TSVInfo` struct and simply call `std::cout << it;` there.
What do you think?

Reply:
Yes, please write new code in the "good" format. Then, in a separate PR we pass the clang-format comb on all the files.